### ISSUE TITLE
Error opening configuration file

### ISSUE THREAD

**Issue by Joentje** [Joentje/nordvpn-proxy#4]
When a configuration is not present, it will fail to connect.

You will get an error like this:

```
Options error: In [CMD-LINE]:1: Error opening configuration file: ....nordvpn.com.tcp.ovpn
Use --help for more information.

2019-12-01 17:43:28 INFO: Connection to server: Country #626
2019-12-01 17:43:28 INFO: Current load: 9
2019-12-01 17:43:28 INFO: Info updated at: 2019-12-01 16:36:20
2019-12-01 17:43:28 INFO: Server IP: 1.1.1.1
```

**Comment by Joentje** [comment]
#5

**Comment by Joentje** [comment]
Fixed in #5
